src/keys.rs

```diff
@@ -264,8 +258,9 @@ impl IssuanceAuthorizingKey {
     }
 
     /// Returns the raw bytes of the issuance key.
+    // Note that the unwrap call never fails since the issuance authorizing key is exactly 32 bytes.
```

Comment:
should have `///` to keep the comment structure.
`/// Unwrap call never fails since the issuance authorizing key is exactly 32 bytes.` (no need to Note)

Reply:
Done.